packages/core/src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -3,10 +3,22 @@
   grid-template-columns: var(--table-grid-template-columns);
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 2;
   background-color: inherit;
+  min-width: 100%;
+  width: fit-content;
 
   > * {
     background-color: inherit;
   }
+
+  &.virtualized {
+    overflow: auto;
+    width: auto;
+    scrollbar-width: none;
+
+    &::-webkit-scrollbar {
```

Comment:
We don't want?

Reply:
on virtualized we're now synchronizing scrolls (at least temporarily?), so we don't want the header scrollbar to be shown, the user should feel as if the header moves as one unit with the body